raytracer.cpp

```diff
@@ -81,22 +83,27 @@ Triangles triangles_from_scene(const aiScene* scene) {
 }
 
 
+static const char USAGE[] =
+R"(Usage: raytracer <filename> [options]
+
+Options:
+  -w --width=<px>           Width of the image [default: 640].
+  -a --aspect=<num>         Aspect ratio of the image. If the model has
+                            specified the aspect ratio, it will be used.
+                            Otherwise default value is 1.
```

Comment:
In Blender ist dieser Wert immer 0. Weiss nicht warum genau, das müsste man untersuchen.